src/Admin/CategoryAdmin.php

```diff
@@ -30,6 +32,23 @@ class CategoryAdmin extends ContextAwareAdmin
         'cascade_validation' => true,
     ];
 
+    /**
+     * @var CategoryManagerInterface
+     */
+    protected $categoryManager;
+
+    public function __construct(
```

Comment:
Please add PHPDoc where we don't have an explicit type in the method/constructor signature